src/api/github-client.ts

```diff
@@ -237,4 +250,21 @@ export class CheGithubClient {
 
     return sortedSemanticTags.map(tag => tag.data)
   }
+
+  private async getCommitData(repo: string, commitId: string) {
```

Comment:
Minor, Could you apply return type, please?

Reply:
Usually I always add return type, but this is an exception from my rules. The problem is that return type is a generic defined by octokit GH client lib. I don't want to write a huge interface myself  (and more problematic to support it then) nor mess up with types of the lib, especially because the function is private for the client class (here TS lang server works fine and for development the type is calculated, so even autocomplete works).